src/main/java/frc/team2412/robot/subsystem/DrivebaseSubsystem.java

```diff
@@ -151,6 +155,16 @@ public DrivebaseSubsystem() {
                 BACK_RIGHT_CONFIG.create(supportAbsoluteEncoder) };
         moduleMaxVelocityMetersPerSec = MODULE_MAX_VELOCITY_METERS_PER_SEC;
 
+        if (DRIVER_VIS_ENABLED) {
+            UsbCamera driverCamera = new UsbCamera("Driver Vision Front", Hardware.FRONT_CAM);
+            driverCamera.setResolution(160, 90);
+            CameraServer.addCamera(driverCamera);
+            CameraServer.startAutomaticCapture();
+            tab.add("Driver camera", driverCamera)
+                    .withPosition(5, 0)
+                    .withSize(2, 2);
```

Comment:
Small nit: the camera feed is quite small at 2x2 widget size and is far more readable at 4x4, so increasing the default size to that would be nice